docs/tutorial/using-selenium-and-webdriver.md

```diff
@@ -86,12 +86,12 @@ const driver = new webdriver.Builder()
   // The "9515" is the port opened by chrome driver.
   .usingServer('http://localhost:9515')
   .withCapabilities({
-    chromeOptions: {
+    'goog:chromeOptions': {
```

Comment:
Was `goog:chromeOptions` the correct key all along, or did it change? If it changed, is this the correct key for Electron 13?